spec/src/main/asciidoc/queries.asciidoc

```diff
@@ -116,5 +116,23 @@ type Query {
   ...
 ----
 
-==== Unnamed queries
+==== Void Queries
 
+By it's very nature, query methods must return some value, thus it is considered a deployment error for a method with a
```

Comment:
Should we throw a specific error ? Or just RuntimeException ?

Reply:
Good question... In my SPQR fork, I threw an IllegalArgumentException - but that is caught and handled by my ServletContainerInitializer, which then throws a ServletException - and that prevents the app from starting.  I think this will probably need to be vendor-specific, but the net result should be the same - that the app fails to start.  And there _should_ be some message indicating why.  But this is definitely difficult to test for in the TCK...